src/components/ItaliaTheme/Blocks/Listing/CardWithImage/CardWithImageDefault.jsx

```diff
@@ -63,16 +64,17 @@ const CardWithImageDefault = (props) => {
 
   const image = ListingImage({ item, showTitleAttr: false });
```

Comment:
contentHasImage, non avrebbe senso calcolarlo dentro al componente ListingImage?

già alla riga 66 si testava di far vedere l'immagine solo se image è !=null

Se facessimo in questo modo la condizione andrebbe messa solo dentro a ListingImage